please.sh

```diff
@@ -4470,11 +4470,12 @@ publish_sdk () { #
 	git --git-dir="$sdk64"/usr/src/build-extra/.git push origin "$tag"
 }
 
-create_sdk_artifact () { # [--out=<directory>] [--git-sdk=<directory>] [--bitness=(32|64|auto)] [--force] <name>
+create_sdk_artifact () { # [--out=<directory>] [--git-sdk=<directory>] [--architecture=(amd64|x86|arm64)] [--bitness=(32|64)] [--force] <name>
 	git_sdk_path=/
 	output_path=
 	force=
-	bitness=auto
+	architecture=
+	bitness=
```

Comment:
I'm not seeing any places that are relying on `bitness=auto` (including `setup-git-for-windows-sdk`), so I think we're good to explicitly require `--bitness` or `--architecture` to be set. Please let me know if that's not the case.

Reply:
I thought my manual runs were relying on this, allowing me to be lazy and not specify anything but the `<name>` ;-) But I guess I broke it long ago. So we're good to leave `--bitness=auto` behind.